src/DependencyInjection/Configuration.php

```diff
@@ -2,46 +2,52 @@
 
 namespace Symfony\Cmf\Bundle\SonataAdminIntegrationBundle\DependencyInjection;
 
+use Symfony\Component\Config\Definition\Builder\ArrayNodeDefinition;
 use Symfony\Component\Config\Definition\Builder\NodeBuilder;
 use Symfony\Component\Config\Definition\Builder\TreeBuilder;
 use Symfony\Component\Config\Definition\ConfigurationInterface;
+use Symfony\Cmf\Bundle\SonataAdminIntegrationBundle\DependencyInjection\Factory\AdminFactoryInterface;
 
 /**
  * @author Maximilian Berghoff <Maximilian.Berghoff@mayflower.de>
  */
 class Configuration implements ConfigurationInterface
 {
+    /**
+     * @var AdminFactoryInterface[]
+     */
+    private $factories;
+
+    public function __construct(array $factories = [])
+    {
+        $this->factories = $factories;
+    }
 
     /**
      * {@inheritdoc}
      */
     public function getConfigTreeBuilder()
     {
         $treeBuilder = new TreeBuilder();
-        $treeBuilder->root('cmf_sonata_admin_integration')
-            ->fixXmlConfig('bundle')
-            ->children()
-                ->arrayNode('bundles')
-                    ->useAttributeAsKey('name')
-                    ->prototype('array')
-                        ->addDefaultsIfNotSet()
-                        ->beforeNormalization()
-                            ->ifTrue(function ($v) { return is_scalar($v); })
-                            ->then(function ($v) {
-                                return ['enabled' => $v];
-                            })
-                        ->end()
-                        ->children()
-                            ->enumNode('enabled')
-                            ->values([true, false, 'auto'])
-                            ->defaultValue('auto')
-                            ->end()
-                            ->scalarNode('form_group')->defaultValue('form.group_seo')->end()
-                        ->end()
-                    ->end()
-                ->end()
-            ->end();
+        $root = $treeBuilder->root('cmf_sonata_admin_integration');
+
+        $this->addBundlesSection($root);
 
         return $treeBuilder;
     }
+
+    private function addBundlesSection(ArrayNodeDefinition $root)
+    {
+        $bundles = $root->children()->arrayNode('bundles')->children();
```

Comment:
is this for forward compatibility to namespace everything about admins for other bundles under a `bundles` key? not sure we need it, but does not cost much and is the defensive option so we can add things if we ever need to. in short: i agree with this structure ;-)